apis/python/src/tiledb/vector_search/flat_index.py

```diff
@@ -196,7 +195,7 @@ def create(
         ids_array_rows_dim = tiledb.Dim(
             name="rows",
             domain=(0, MAX_INT32),
-            tile=tile_size,
+            tile=int(TILE_SIZE_BYTES / np.dtype(np.uint64).itemsize / dimensions),
```

Comment:
I think this should be

```
int(TILE_SIZE_BYTES / np.dtype(np.uint64).itemsize)
```
as IDs are not a vector but just one value per vector.

However, the main point of having the same `tile_size` between vectors and IDs is that we always read these arrays with the same slicing queries which are mainly filtering different ivf partitions. Having much more values inside ID tiles can give some optimization in disk space but it might lead in suboptimal read filtering by always over-fetching data on the ID array read.

Reply:
Okay awesome talked offline - understand now and don't think we need to make this change.